recipes/regtools/meta.yaml

```diff
@@ -0,0 +1,25 @@
+{% set version = "0.3.0" %}
+package:
+  name: regtools
+  version: {{ version }}
+
+source:
+  fn: "{{ version }}.tar.gz"
+  md5: c2af56b0350b6f2b88e5ce52ab5b8721
+  url: https://github.com/griffithlab/regtools/archive/{{ version }}.tar.gz
+
+requirements:
+  build:
+  - gcc  # [not osx]
+  - llvm # [osx]
+  - cmake
+  run:
+  - libgcc # [not osx]
+about:
+  home: https://github.com/griffithlab/regtools
+  license: MIT
+  summary: "Tools that integrate DNA-seq and RNA-seq data to help interpret mutations in a regulatory and splicing context."
+
+test:
+  commands:
+  - which regtools
```

Comment:
Do you have a better test for this tool? Is `regtools --help` or `--version` working?